carbonmark-api/src/routes/users/[walletOrHandle]/get.ts

```diff
@@ -91,7 +92,10 @@ const handler = (fastify: FastifyInstance) =>
           };
         }) || [];
 
-      const listings = user?.listings?.map(formatListing) || [];
+      const listings = getActiveListings(
```

Comment:
Just a note of caution:
We want to make sure we can still return unexpired listings or those with token dust, that way users can still manage their own listings.
At first glance it looks OK. Maybe we should check if we have a test case to cover these requirements just in case

Reply:
Yep, I created a test case on my account. It made sure that:
 - I could see my dust listing.
 - Other users could not see it (Done at UI level in 071e9f6bf)